tests/conftest.py

```diff
@@ -105,5 +108,45 @@ def dump_user(user):
     db.session.commit()
     return {
         'user1': dump_user(user1),
-        'user2': dump_user(user2)
+        'user1_raw': user1,
+        'user2': dump_user(user2),
+        'user2_raw': user2,
```

Comment:
i think the `user1: dump_user(user1)` are never actually used, should we remove them instead?

Reply:
Yes, if not used, then remove :-)